app/components/CommentForm/CommentForm.css

```diff
@@ -7,6 +7,5 @@
 }
 
 .submitButton:disabled {
-  opacity: 0;
-  transition: opacity var(--easing-medium);
+  display: none;
```

Comment:
I like the fading though 😢

Reply:
Yeah, but I'm not a huge fan of having the button still be there, only invisible.

Reply:
hehe I guess so